templates/layout.html

```diff
@@ -79,126 +79,34 @@
         </ul>
       </div>
 
-      <!-- Collect the nav links, forms, and other content for toggling -->
+      {{!-- Collect the nav links, forms, and other content for toggling --}}
       <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
         <ul class="nav navbar-nav navbar-right">
           {{#if currentUser}}
-            <li class="{{#if isActiveRoute 'newCompetition'}}active{{/if}} leaf">
-              <a href="{{pathFor 'newCompetition'}}"
-                 data-toggle="tooltip" data-placement="bottom" data-container="body"
-                 title="New competition">
-                <span class="glyphicon glyphicon-plus"></span>
-              </a>
-            </li>
+            {{#with newCompetitionTab}}
+              {{> oneTab}}
```

Comment:
I think this can become:

```
{{> oneTab newCompetitionTab}}
```

(see "Template includes" on https://www.discovermeteor.com/blog/a-guide-to-meteor-templates-data-contexts/) Can't find a good explanation of this on the meteor docs.